code/modules/mob/living/carbon/alien/larva/larva_defense.dm

```diff
@@ -2,28 +2,13 @@
 
 /mob/living/carbon/alien/larva/attack_hand(mob/living/carbon/human/M)
 	if(..())
-		var/damage = rand(1, 9)
-		if (prob(90))
-			playsound(loc, "punch", 25, 1, -1)
-			log_combat(M, src, "attacked")
-			visible_message("<span class='danger'>[M] kicks [src]!</span>", \
-					"<span class='userdanger'>[M] kicks you!</span>", null, COMBAT_MESSAGE_RANGE)
-			if ((stat != DEAD) && (damage > 4.9))
-				Unconscious(rand(100,200))
-
-			var/obj/item/bodypart/affecting = get_bodypart(ran_zone(M.zone_selected))
-			apply_damage(damage, BRUTE, affecting)
-		else
-			playsound(loc, 'sound/weapons/punchmiss.ogg', 25, 1, -1)
-			visible_message("<span class='danger'>[M]'s kick misses [src]!</span>", \
-					"<span class='userdanger'>[M]'s kick misses you!</span>", null, COMBAT_MESSAGE_RANGE)
-
```

Comment:
Again, removing the probs is pretty irrelevant to this PR, you should add your changes and thats it rather than make big changes to the existing code.